doc/doxygen/headers/automatic_and_symbolic_differentiation.h

```diff
@@ -20,8 +20,34 @@
  * to automatic and symbolic differentiation.
  *
  * Below we provide a very brief introduction as to what automatic and symbolic differentiation are,
- * what variations of these computational / numerical schemes exist, and how they are integrated
- * within deal.II's framework.
+ * what variations of these computational/numerical schemes exist, and how they are integrated
+ * within deal.II's framework. The purpose of all of these schemes is to automatically compute
+ * (approximations of) the derivative of functions in cases where one does not want to
```

Comment:
I would prefer this to read something like

> The purpose of all of these schemes is to automatically compute the derivative of functions, or approximations thereof, ...

As is stated later, well-formulated AD method (and, by nature, correctly implemented SD methods) may return exact derivatives to machine precision for first and sometimes higher-order derivatives.